src/test/java/Tests.java

```diff
@@ -0,0 +1,103 @@
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.jupiter.api.Assertions;
+import parser.JsonParser;
+import parser.NoSuchFileException;
+import shop.Cart;
+import shop.RealItem;
+import shop.VirtualItem;
+import java.io.File;
+import java.math.BigDecimal;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+public class Tests {
+    @Test
+    public void testJSonParserWriteToFile(){
+
+            JsonParser jsonParser = new JsonParser();
+            jsonParser.writeToFile(new Cart("CartItem"));
+            Path path = Paths.get("src/main/resources/CartItem.json");
+            Assertions.assertTrue(Files.exists(path), String.format("File %s doesn't exist", path));
+    }
+
+    @Test
+    public void testJSonParserReadFromFile(){
+        JsonParser jsonParser = new JsonParser();
+        Assertions.assertNotNull(jsonParser.readFromFile(new File("src/main/resources/CartItem.json")));
+    }
+
+    @Test
+    public void testRealItem(){
+        BigDecimal expectedResult = new BigDecimal(200);
+        RealItem realItem = new RealItem();
+
+        realItem.setWeight(200);
+        Assertions.assertEquals(expectedResult,realItem.getWeight() , String.format("Expected weight was %s, but actual was %s", expectedResult, realItem.getWeight()));
+    }
+
+    @Test
+    public void testVirtualItem(){
+        BigDecimal expectedResult = new BigDecimal(25.5);
+        VirtualItem virtualItem = new VirtualItem();
+
+        virtualItem.setSizeOnDisk(25.5);
+        Assertions.assertEquals(virtualItem.getSizeOnDisk(), expectedResult, String.format("Expected size was %s, but actual was %s", expectedResult, virtualItem.getSizeOnDisk()));
+    }
+
+    @Test
+    public void testCart1(){
```

Comment:
what are you testing here? please provide me some more information about it in the method name